core-tests/shared/src/test/scala/zio/FiberRefSpec.scala

```diff
@@ -1,187 +1,186 @@
 package zio
 
+import zio.FiberRefSpecUtil._
 import zio.duration.Duration
-
-class FiberRefSpec extends BaseCrossPlatformSpec {
-  override def is = "FiberRefSpec".title ^ s2"""
-   Create a new FiberRef with a specified value and check if:
-      `get` returns the current value.                         $e1
-      `get` returns the current value for a child.             $e2
-      `set` updates the current value.                         $e3
-      `set` by a child doesn't update parent's value.          $e4
-      `locally` restores original value.                       $e5
-      `locally` restores parent's value.                       $e6
-      `locally` restores undefined value.                      $e7
-      its value is inherited on join.                          $e8
-      initial value is always available.                       $e9
-      `update` changes value.                                  $e10
-      `updateSome` changes value.                              $e11
-      `updateSome` not changes value.                          $e12
-      `modify` changes value.                                  $e13
-      `modifySome` not changes value.                          $e14
-      its value is inherited after simple race.                $e15
-      its value is inherited after a race with a bad winner.   $e16
-      its value is not inherited after a race of losers.       $e17
-      the value of the looser is inherited in zipPar           $e18
-      nothing gets inherited with a failure in zipPar          $e19
-    """
-
+import zio.test.Assertion._
+import zio.test._
+import zio.test.mock.MockClock
+import zio.duration._
+
+object FiberRefSpec
+    extends ZIOBaseSpec(
+      suite("Create a new FiberRef with a specified value and check if:")(
```

Comment:
Can you put this in a root suite called `FiberRefSpec`? Just to be consistent with other tests.